### PR TITLE
Add policy AC-K8-NS-SE-M-0188 for CVE-2020-8554

### PR DESCRIPTION
**Pull request by gauravgogia-accurics**

### CODE REVIEW

File: pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[service.id] {
+    service := input.{{.resource_type}}[_]
+    service.config.kind == "Service"
```
Comment (williepaul) [inline pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego:5]: Looks like this check is unnecessary--kubernetes_service resource_type implies this kind

File: pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    type_check(service.config.spec)
```
Comment (williepaul) [inline pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego:6]: IMO we should keep our checks as simple as possible. it looks like you're just trying to do the following:
service.config.spec.type == "ClusterIP"
not service.config.spec.type
not service.config.spec.externalIPs

Do we also need a check for service.config.spec.clusterIPs?

File: pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego (hunk 1 above)
Comment (gauravgogia-accurics) [reply pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego:5]: I see, will remove that check

File: pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego (hunk 2 above)
Comment (gauravgogia-accurics) [reply pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego:6]: Service type may or may not be set. If its set then I want to if its **ClusterIP**

**Review by williepaul — APPROVED** [review]

**Pull request merged**